cryptographic_estimators/MAYOEstimator/MAYOAlgorithms/direct_attack.py

```diff
@@ -34,8 +35,8 @@ class DirectAttack(MAYOAlgorithm):
 
     INPUT:
 
-    - ``problem`` -- DummyProblem object including all necessary parameters
-    - ``w`` -- linear algebra constant (default: Obtained from MAYOAlgorithm)
+    - ``problem`` -- MAYOProblem object including all necessary parameters
+    - ``w_mq`` -- linear algebra constant (default: Obtained from MAYOAlgorithm)
```

Comment:
Since it is called `w` all the other algorithms I suggest to leave it as `w` only